cernopendata/modules/fixtures/data/records/cms-tools-vm-image-2012.json

```diff
@@ -0,0 +1,63 @@
+[
+{
+  "abstract": {
+      "description": "The scripts provided create the CMS virtual machine image based on <a href=\"https://cernvm.cern.ch\">CernVM</a>, with Scientific Linux CERN 6 (slc6)."
+  }, 
+  "accelerator": "CERN-LHC", 
+  "authors": [
+      {
+          "name": "Blomer, Jakob"
+      }
+  ], 
+  "collections": [
+      "CMS-Tools"
+  ], 
+  "date_created": "2012", 
+  "date_published": "2017", 
+  "distribution": {
+      "formats": [
+          "txt"
+      ],
+      "number_files": 3,
+      "size": "FIXME"
+  }, 
+  "experiment": "CMS", 
+  "files": [
```

Comment:
Here is the file information:
```
{
  "files": [
    {
      "checksum": "adler32:234a36f3",
      "size": 178,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/cernvm-script.txt"
    },
    {
      "checksum": "adler32:f223d6d8",
      "size": 3071,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/cms-user-data.txt"
    },
    {
      "checksum": "adler32:786f33c7",
      "size": 152,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/opendata-desktop-settings"
    }
  ]
}
```

Note that I create 2012 directory, as we don't have versions in 2010 one...